plugins/inventory/hrobot.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is BSD licensed.
+# Modules you write using this snippet, which is embedded dynamically by Ansible
+# still belong to the author of the module, and may assign their own license
+# to the complete work.
+#
+# Copyright (c), Felix Fontein <felix@fontein.de>, 2019
```

Comment:
You need to change this line :D

Also, please use the GPLv3+ (as for all other plugins, except module_utils). Use the short form like here:
https://github.com/ansible-collections/community.hrobot/blob/main/plugins/modules/failover_ip.py#L4-L5